src/action.js

```diff
@@ -29,7 +29,28 @@ async function exportSecrets() {
 
     const defaultOptions = {
         prefixUrl: vaultUrl,
-        headers: {}
+        headers: {},
+        https: {}
+    }
+
+    const tlsSkipVerify = core.getInput('tlsSkipVerify', { required: false }) != 'false';
```

Comment:
What if someone sets this to `False`? Seems potentially a bit fragile if I'm correct that it's case-sensitive.

Reply:
Casting to lower to make this a bit less fragile.